src/CMDParse.cpp

```diff
@@ -0,0 +1,57 @@
+#include "CMDParse.h"
+
+EERAModel::ArgumentParser::ArgumentParser(int argc, char** argv)
+{
+    try
+	{
+		TCLAP::CmdLine cmd("A model structure argument and mode of running must be provided",   // Help string message
```

Comment:
Indentation is all over the place. Please tidy it up.

Reply:
It is only like this on GitHub, on VSCode it is aligned for me. Any ideas how to handle this?

Reply:
On VSCode, try View->Render Whitespace. It may show that some of your alignment is done with tabs and some with spaces, which could account for why it looks different. In general, we should use spaces for everything to try and keep things consistent.

Reply:
I already have that turned on and it shows all the indentation to be spaces only.